Apps/W1/HybridSL/test/src/SLBCGenProdPostingGroup.XmlPost.al

```diff
@@ -5,7 +5,6 @@
 namespace Microsoft.DataMigration.SL;
 
 using Microsoft.Finance.GeneralLedger.Setup;
```

Comment:
Could you maybe revert the changes in the two test files? I tried to fix it myself, but I didn't succeed 😋 No changes should be needed in those two.

Reply:
Or just leave it :-) I simply won't port these changes when creating the internal PR (which I'll do right away)!